SPEC/OBJECT.md

```diff
@@ -198,7 +198,8 @@ A great source of [examples][] can be found in the tests for this API.
 
 `options` is a optional argument of type object, that can contain the following properties:
 
-- `enc`, the encoding of multihash (base58, base64, etc), if any.
+- `enc`, The encoding type the output should beencoded with (json, xml, or text), if any.
```

Comment:
I think this should just be removed - it's not documented on docs.ipfs.io and the js-ipfs http and cli do no respect it. I think it's a hangover from pre-CID days.

It's not the same as the `encoding` flag: https://docs.ipfs.io/reference/api/http/#flags